llama_index/llms/litellm_utils.py

```diff
@@ -0,0 +1,260 @@
+import logging
+import os
+import re
+from typing import Any, Callable, Dict, List, Optional, Sequence, Type, Union
+
+import litellm
+from litellm import completion, acompletion
```

Comment:
This is causing the unit tests to fail -- should probably move it to the functions that use these imports